core/modules/layout/tests/layout.test

```diff
@@ -3414,3 +3414,39 @@ class LayoutFlexibleTemplateTest extends BackdropWebTestCase {
   }
 
 }
+
+/**
+ * Tests that default layout paths are handled properly.
+ */
+class LayoutPathTest extends BackdropWebTestCase {
+  protected $profile = 'testing';
+  protected $admin_user;
+
+  function setUp() {
+    parent::setUp(array('layout', 'layout_test'));
```

Comment:
I don't think you have to enable the layout module explicitly, as it's a layout test, anyway. It should be sufficient to enable layout_test.

Reply:
Most of the other `parent::setUp()` calls in this file include `'layout'`. Should they all be changed?

Reply:
Not in this issue. :wink: